assertk-common/src/main/kotlin/assertk/assertions/map.kt

```diff
@@ -84,8 +84,7 @@ fun <K, V> Assert<Map<K, V>>.contains(element: Pair<K, V>) {
 @PlatformName("mapContainsAll")
 fun <K, V> Assert<Map<K, V>>.containsAll(vararg elements: Pair<K, V>) {
     if (elements.all { (k, v) -> actual[k] == v }) return
-    val notFound = elements.filterNot { (k, v) -> actual[k] == v }
-    expected("to contain all:${show(elements.toMap())} some elements were not found:${show(notFound.toMap())}")
+    expected("to contain all:${show(elements.toMap())} but was:${show(actual)}")
```

Comment:
Hm, this just prints out both maps, think we can do better and show what's different.

Reply:
@evant Changed the assertion message to print expected map, actual map and what's different.